### PR TITLE
Rename algorand-builder to algo-builder

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

**Review by man-of-code — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# algorand-builder-templates
+# algo-builder-templates
```
Comment (robert-zaremba) [inline README.md]:
```suggestion
# Algo Builder Templates
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-Dapps templates for [Algorand Builder](https://github.com/scale-it/algorand-builder). Use these templates to quickly setup/get started with your Dapp implementation.
+Dapps templates for [Algorand Builder](https://github.com/scale-it/algo-builder). Use these templates to quickly setup/get started with your Dapp implementation.
```
Comment (robert-zaremba) [inline README.md]:
```suggestion
Dapps templates for [Algo Builder](https://github.com/scale-it/algo-builder). Use these templates to quickly setup/get started with your Dapp implementation.
```

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 ## Usage
 Each template is a single project (with 1 package.json file) comprising of the following:
- - An `algob` project : Used to deploy ASA's, stateless and stateful contracts using algob `deployer` object. Use `algob deploy` to deploy your `/scripts`. Sample algob project can  be found [here](https://github.com/scale-it/algorand-builder/tree/master/packages/algob/sample-project), check the project readme.md for more details.
-   Read more about `@algorand-builder/algob` [here](https://github.com/scale-it/algorand-builder/tree/master/packages/algob).
+ - An `algob` project : Used to deploy ASA's, stateless and stateful contracts using algob `deployer` object. Use `algob deploy` to deploy your `/scripts`. Sample algob project can  be found [here](https://github.com/scale-it/algo-builder/tree/master/packages/algob/sample-project), check the project readme.md for more details.
+   Read more about `@algo-builder/algob` [here](https://github.com/scale-it/algo-builder/tree/master/packages/algob).
```
Comment (robert-zaremba) [inline README.md:8]: Maybe we should link to the root? What do you think?

**Review by robert-zaremba — APPROVED** [review]
Left few suggestions.

File: README.md (hunk 3 above)
Comment (ratik21) [reply README.md:8]: Yes, good idea. But we have already linked root package in the main header. Here - [https://github.com/scale-it/algo-builder-templates/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3](https://github.com/scale-it/algo-builder-templates/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)

**Pull request merged**